lib/percy/config.rb

```diff
@@ -42,7 +42,7 @@ def debug
     end
 
     def repo
-      @repo ||= Percy::Client::Environment.repo
```

Comment:
It would be great to have a little historical context on how this works because this feels like a substantial change?